tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -448,6 +448,31 @@ public function theNumberOfVersionsShouldBe($number) {
 		);
 	}
 
+	/**
+	 * @Then the number of etag element in the response should be :arg1
```

Comment:
```suggestion
	 * @Then the number of etag element in the response should be :number
```
If you wish you can change it to something meaningful like this
else LGTM :+1: